brave/browser-laptop
# handle sites that check navigator.plugins

allow adobe to auto-detect install
todo - add flash enable message box to adobe install page when flash is installed
todo - reload page after enabling flash for pages that detect navigation.plugins
partial fix for #6306


Test Plan:
1. Go to Pandora.com
2. Click on adobe flash link
3. Install flash
4. Enable flash
5. Go to Pandora.com
6. Click on adobe flash link
7. Message box should open
8. Deny
9. Reload Pandora.com
10. Click on adobe flash link
11. Allow
12. Reload Pandora.com 

Flash should be enabled

cc @diracdeltas
